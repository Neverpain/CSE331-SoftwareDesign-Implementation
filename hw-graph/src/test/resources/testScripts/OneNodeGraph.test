# Tests that a graph can be created, a node can be
# added to it, and a node can be printed out

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Prints the nodes contained in the graph
ListNodes graph1