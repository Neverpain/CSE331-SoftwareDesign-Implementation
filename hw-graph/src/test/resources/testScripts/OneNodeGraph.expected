# Tests that a graph can be created, a node can be
# added to it, and a node can be printed out

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Prints the nodes contained in the graph
graph1 contains: n1